generator/transit_generator.cpp

```diff
@@ -319,7 +256,182 @@ void BuildTransit(string const & mwmDir, string const & countryId,
   w.Seek(startOffset);
   header.Visit(serializer);
   w.Seek(endOffset);
+
   LOG(LINFO, (TRANSIT_FILE_TAG, "section is ready. Header:", header));
 }
+
+void GraphData::Append(GraphData const & rhs)
```

Comment:
s/Append/AppendTo/

I would expect that 
* ```a.Append(b)``` appends ```b``` to ```a```.
* ```a.AppendTo(b)``` appends ```a``` to ```b```.

Reply:
Done